src/subtype.c

```diff
@@ -656,7 +658,9 @@ static int var_gt(jl_tvar_t *b, jl_value_t *a, jl_stenv_t *e, int param)
     }
     if (!((bb->ub == (jl_value_t*)jl_any_type && !jl_is_type(a) && !jl_is_typevar(a)) || subtype_ccheck(a, bb->ub, e)))
         return 0;
-    bb->lb = simple_join(bb->lb, a);
+    jl_value_t *lb = simple_join(bb->lb, a);
+    if (!e->intersection || !subtype_by_bounds(lb, b, e))
```

Comment:
You've angered analyzegc here:
```
/cache/build/amdci5-2/julialang/julia-master/src/subtype.c:662:30: error: Passing non-rooted value as argument to function that may GC [julia.GCChecker]
    if (!e->intersection || !subtype_by_bounds(lb, b, e))
```

since `subtype_by_bounds` is not marked JL_NOTSAFEPOINT, nor is `simple_join` marked `JL_PROPAGATES_ROOT`